src/app/pages/signing/signing.component.html

```diff
@@ -1,74 +1,115 @@
+        <label for="signing-box">
+          <b>Type your message to [friend name] below.</b>
+        </label>
+      </div>
+      <div class="row">
+        <textarea #signingInputArea id="signing-box" class="form-control" autofocus
+          (keyup)='updateCount(signingInputArea)' [(ngModel)]='signingBoxText'
+          [ngStyle]="{'font-family': currentFont}"></textarea>
+        <span id="char-count-display" *ngIf="charCount >= 2000" [ngStyle]="{'color': charCountColor}">
+          {{ charCount }} / {{ maxCharCount }}
+        </span>
+      </div>
     </div>
-    <div class="row">
-      <markdown emoji [ngStyle]="{'background-color': backgroundColor, 'color': textColor, 'font-family': currentFont}"
-        id="preview-text" data="{{ signingBoxText }}"></markdown>
+
+    <!-- Formatting Tools -->
+    <div class='col-md-12 order-md-3'>
+      <div class=" row justify-content-center" id="options">
+        
+        <!-- Font dropdown -->
+        <div class="dropdown">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="font-selector-dropdown"
+            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            {{ currentFontDisplay }}
+          </button>
+          <div class="dropdown-menu" aria-labelledby="font-selector-dropdown">
+            <!-- javascript: void(0) does nothing, but the href makes the cursor become a clicky thing on hover -->
+            <a class="dropdown-item" style="font-family: Arial, sans-serif" (click)="selectFont('Arial, sans-serif')"
+              href="javascript: void(0)">Arial</a>
+            <a class="dropdown-item" style="font-family: Comic Sans MS, sans-serif"
+              (click)="selectFont('Comic Sans MS, sans-serif')" href="javascript: void(0)">Comic Sans MS</a>
+            <a class="dropdown-item" style="font-family: Courier New, monospace"
+              (click)="selectFont('Courier New, monospace')" href="javascript: void(0)">Courier New</a>
+            <a class="dropdown-item" style="font-family: Dancing Script, cursive"
+              (click)="selectFont('Dancing Script, cursive')" href="javascript: void(0)">Dancing Script</a>
+            <a class="dropdown-item" style="font-family: Montserrat, sans-serif"
+              (click)="selectFont('Montserrat, sans-serif')" href="javascript: void(0)">Montserrat</a>
+            <a class="dropdown-item" style="font-family: Roboto, sans-serif" (click)="selectFont('Roboto, sans-serif')"
+              href="javascript: void(0)">Roboto</a>
+            <a class="dropdown-item" style="font-family: Times New Roman, serif"
+              (click)="selectFont('Times New Roman, serif')" href="javascript: void(0)">Times New
+              Roman</a>
+          </div>
+        </div>
+
+        <!-- Formatting buttons -->
+        <button type="button" (click)="addFormatting(signingInputArea, '**')" class="btn btn-secondary btn-format mr-2"
+          id="button-bold" #b>
```

Comment:
Remove unused variable
```suggestion
          id="button-bold">
```